source/js/src/video.js.es6

```diff
@@ -1,52 +1,100 @@
 //= require "../lib/require.2.3.6.min.js"
 
-// NOTE: We use require.js to load Vimeo, then initialize Vimeo.
-// NOTE: …/player.js is preloaded in the <head>!
-require(['https://player.vimeo.com/api/player.js'], function(Player) {
+// Videos
+;(function() {
+  let loaded = false;
   let initialized = false;
+  const videos = [];
 
-  function initialize () {
-    if (initialized) return;
+  /**
+   * Load embedly JS and initialize default settings
+   * @returns {Promise<unknown>}
+   */
+  const loadEmbedly = function(staticVideoElements) {
+    return new Promise((resolve, reject) => {
+      if (loaded) resolve(videos);
 
-    const vimeoSrc = document.querySelector('[data-vimeo-url], [data-vimeo-id]');
-    if (!vimeoSrc) return false;
-    initialized = true;
+      if (staticVideoElements && staticVideoElements.length) {
+        for (let i = 0; i < staticVideoElements.length; i++) {
```

Comment:
95% sure `for …` is cross-browser compatible.  But this is a `NodeArray` or whatever it's called…so worth having it tested in IE11.

This seems to be aware of that 😉 given you didn't do `staticVideoElements.forEach`.

Reply:
Precisely 👌